### PR TITLE
[16.0][MIG] web_dialog_size

### PR DESCRIPTION
**Pull request by tarteo**
Works but uses legacy code of Odoo.

### CODE REVIEW

File: web_dialog_size/readme/CONFIGURATION.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+By default, the module respects the caller's ``dialog_size`` option.
```
Comment (flotho) [inline web_dialog_size/readme/CONFIGURATION.rst]: duplicate configuration file

File: web_dialog_size/readme/CONFIGURE.rst [hunk 2]
```diff
@@ -0,0 +1,5 @@
+If you want to set dialog boxes maximized by default, you need to:
```
Comment (flotho) [inline web_dialog_size/readme/CONFIGURE.rst:2]: duplicate configuration file

**Review by flotho — CHANGES_REQUESTED** [review]
Hi, 

thanks for the job already done.

tested on runboat and doesn't seems successful :
set up done:
image

yet dialog size still the same with no icons : 

image

**Comment by tarteo** [comment]
@flotho Hey, thanks for you review. I fixed your issues!

**Comment by tarteo** [comment]
Failed checks are unrelated to the commit. Let me re-run them let see if it fixes stuff.

File: web_dialog_size/readme/CONFIGURE.rst [hunk 3]
```diff
@@ -1,4 +1,5 @@
-If you want to set dialog boxes maximized by default, you need to:
+By default, the module respects the caller's ``dialog_size`` option.
+If you want to set d  ialog boxes maximized by default, you need to:
```
Comment (astirpe) [inline web_dialog_size/readme/CONFIGURE.rst]: Typo: `d  ialog`

**Review by astirpe — CHANGES_REQUESTED** [review]

**Review by flotho — APPROVED** [review]
tested on runbot :
image

functionnal OK

LGTM

**Review by fcvalgar — APPROVED** [review]
LGTM. thak you @tarteo

**Review by rafaelbn — CHANGES_REQUESTED** [review]
Fails in my test

https://www.loom.com/share/441aa9d471564e308a094cd277471603

```
Odoo Client Error

An error occurred

Please use the copy button to report the error to your support service.
Odoo Client Error

An error occurred

Please use the copy button to report the error to your support service.
UncaughtClientError > TypeError

Uncaught Javascript Error > browser.navigator.clipboard is undefined

TypeError: browser.navigator.clipboard is undefined
    onClickClipboard@http://oca-web-16-0-pr2425-52b44c281b9b.runboat.odoo-community.org/web/assets/448-03d15d3/web.assets_backend.min.js:619:20
    mainEventHandler@http://oca-web-16-0-pr2425-52b44c281b9b.runboat.odoo-community.org/web/assets/447-3f55f6a/web.assets_common.min.js:2085:77
    listener@http://oca-web-16-0-pr2425-52b44c281b9b.runboat.odoo-community.org/web/assets/447-3f55f6a/web.assets_common.min.js:1273:15

Odoo Client Error

An error occurred

Please use the copy button to report the error to your support service.
UncaughtClientError > TypeError

Uncaught Javascript Error > browser.navigator.clipboard is undefined

TypeError: browser.navigator.clipboard is undefined
    onClickClipboard@http://oca-web-16-0-pr2425-52b44c281b9b.runboat.odoo-community.org/web/assets/448-03d15d3/web.assets_backend.min.js:619:20
    mainEventHandler@http://oca-web-16-0-pr2425-52b44c281b9b.runboat.odoo-community.org/web/assets/447-3f55f6a/web.assets_common.min.js:2085:77
    listener@http://oca-web-16-0-pr2425-52b44c281b9b.runboat.odoo-community.org/web/assets/447-3f55f6a/web.assets_common.min.js:1273:15

Odoo Client Error

An error occurred

Please use the copy button to report the error to your support service.
UncaughtClientError > TypeError

Uncaught Javascript Error > browser.navigator.clipboard is undefined

TypeError: browser.navigator.clipboard is undefined
    onClickClipboard@http://oca-web-16-0-pr2425-52b44c281b9b.runboat.odoo-community.org/web/assets/448-03d15d3/web.assets_backend.min.js:619:20
    mainEventHandler@http://oca-web-16-0-pr2425-52b44c281b9b.runboat.odoo-community.org/web/assets/447-3f55f6a/web.assets_common.min.js:2085:77
    listener@http://oca-web-16-0-pr2425-52b44c281b9b.runboat.odoo-community.org/web/assets/447-3f55f6a/web.assets_common.min.js:1273:15

TEST
(bb97c4019-f207-4dbc-b48c-d8e0e5cd34d5)
```

**Comment by marielejeune** [comment]
Hi,
Thanks for the work.
The "Search More..." button in m2o default widget is not working anymore. See the error when e.g. trying to fill the country_id of a res.partner
image

**Comment by kikopeiro** [comment]
> Hi, Thanks for the work. The "Search More..." button in m2o default widget is not working anymore. See the error when e.g. trying to fill the country_id of a res.partner image

This bug is the same as @rafaelbn 's [error](https://github.com/OCA/web/pull/2425#pullrequestreview-1366844145)

**Comment by rafaelbn** [comment]
Please @tarteo could you kindly review last contributors review? ❤️ 😄 thank you!

**Comment by rafaelbn** [comment]
I recorded a video https://github.com/OCA/web/pull/2425#pullrequestreview-1366844145 @Tardo , this can help you! 🙏

**Comment by tarteo** [comment]
@rafaelbn Thanks for your rieview I'll look at it rn.

File: web_dialog_size/readme/CONFIGURE.rst (hunk 3 above)
Comment (tarteo) [reply web_dialog_size/readme/CONFIGURE.rst]: Fixed!

**Comment by tarteo** [comment]
@kikopeiro Thanks I fixed this issue.

**Pull request merged**